scripts/qualtrics/update_intake_questions.py

```
@@ -0,0 +1,124 @@
+
+    desired = _work_shift_body()
+
+    logger.info("Fetching current state of %s/%s", INTAKE, WORK_SHIFT_QID)
+    try:
+        current = fetch_question(
+            INTAKE,
+            WORK_SHIFT_QID,
+            cfg.qualtrics_base_url,
+            cfg.qualtrics_api_key,
+        )
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)